scout/server/blueprints/institutes/controllers.py

```diff
@@ -371,6 +374,7 @@ def update_institute_settings(store: MongoAdapter, institute_obj: Dict, form: Mu
         sanger_recipients=get_sanger_recipients(form),
         coverage_cutoff=int(form.get("coverage_cutoff")),
         frequency_cutoff=float(form.get("frequency_cutoff")),
+        show_all_cases_categories=form.getlist("show_all_cases_categories"),
```

Comment:
```suggestion
        show_all_cases_status=form.getlist("show_all_cases_status"),
```

Reply:
Oh, just read the rest and realised you have it as admin only. That makes some sense I guess, but then this would cause the setting cleared every time a non-admin uses it. A suggestion would be to use a conditional, admin aware getter like for Clinvar submitters below.